specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2024-02-01-preview/ThreatIntelligence.json

```
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Sentinel Upload API",
+    "description": "Specification for Microsoft Sentinel Upload API.",
+    "version": "2024-02-01-preview"
+  },
+  "host": "api.ti.sentinel.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/workspaces/{workspaceId}/threatintelligencestixobjects:upload": {
+      "post": {
+        "x-ms-examples": {
+          "Upload Stix Objects": {
+            "$ref": "./examples/threatintelligence/UploadStixObjects.json"
+          }
+        },
+        "summary": "Upload Stix objects to the workspace",
+        "description": "Upload an array of Stix objects to the workspace specified",
```

I recommend that you expand the STIX acronym here, for the benefit of new users (I didn't know what it was -- had to google it).
```suggestion
        "description": "Upload an array of Structured Threat Information Expression (STIX) objects to the workspace specified",
```